packages/ffe-cards/less/ffe-new-card.less

```diff
@@ -0,0 +1,107 @@
+.ffe-new-card {
```

Comment:
One thought on the styling that makes out the basics of this box. Perhaps it should be extracted as a totally separate class? I know people are using custom boxes with the same basics, so it might be good to help facilitate those boxes.
The new React component here could simply apply both classes `.ffe-card-basics` and `.ffe-new-card` (or whatever  we call them).